spec/models/asset_spec.rb

```diff
@@ -232,6 +232,18 @@
       asset = FactoryBot.build(:asset, draft: true, access_limited_user_ids: ['another-user-id'])
       expect(asset).not_to be_accessible_by(user)
     end
+
+    it 'returns true if the asset is draft and access limited and the users org is authorised to view it' do
```

Comment:
Minor, but this feels like it wants a context, but I understand it's consistent with the one above :-).

Reply:
Fixed - refactored tests using contexts